advancedblacklist/abc.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2021 - Jojo#7791
+# Licensed under MIT
+
+from abc import ABC
+
+from redbot.core import Config, commands
```

Comment:
`Config` doesn't seem to be used here. You can type hint it inside the mixin, or remove it entirely if its not referenced as an attribute for `self` in any of this mixin's subclasses.